src/main/webapp/WEB-INF/message-thread.jsp

```diff
@@ -47,9 +47,9 @@
     <div id="message-container">
       <div class="message-div">
         <div class="message-header">
-          User: <%= parent.getUser() %> --
+          User: <a href="/users/<%= messages.get(i).getUser() %>"><%= messages.get(i).getUser() %></a> --
```

Comment:
Why parent.getUser() was changed to messages.get(i).getUser()?

Reply:
Oops! Good catch, I'll change that real quick, I was just copy/pasting without thinking